### DISCUSSION TITLE
multi Circle hold on Sketch

### DISCUSSION THREAD

**Discussion by niletry** [zalo/CascadeStudio#150] [Q&A]
Hi , How to add circle on sketch? there is my simple code, but it not work. what can I do?
```
let faceBuilder = new Sketch([0, 0])
    .LineTo([10, 10]).Fillet(1)
    .LineTo([0, 10]).Fillet(1)
    .End(true).Fillet(1);
faceBuilder.Circle([1, 9], .5, true);
faceBuilder.Circle([8, 9], .5, true);
let face = faceBuilder.Face(false);
```
i I comment line 5, it works.
![image](https://user-images.githubusercontent.com/154701/192130949-cc86fd8b-b121-4157-8deb-9adf7409b871.png)

comment lint 6 , it works.
![image](https://user-images.githubusercontent.com/154701/192130959-5a89da1c-245c-4929-89f8-78d10d82fa3b.png)

**Comment by raydeleu** [comment]
This is a limitation in CascadeStudio. I described this in my CascaceStudioManual in section 8.1.10. Have a look at the discussions #96 to find this manual. Even with one hole you might need to invert the direction of the sketch, but with two or three circles the face is no longer properly closed.
